src/main/java/lottery/domain/LotteryTicket.java

```diff
@@ -17,52 +17,22 @@ public LotteryTicket(List<LotteryNumber> lotteryNumbers) {
             throw new IllegalArgumentException("중복없는 6개의 숫자가 입력되어야 합니다.");
         }
 
-        this.lotteryNumbers = new TreeSet<>();
-//        this.lotteryNumbers = new ArrayList<>();
-        for (LotteryNumber number : lotteryNumbers) {
-            this.lotteryNumbers.add(new LotteryNumber(number.getNumber()));
-        }
-
-//        Collections.sort(this.lotteryNumbers);
+        this.lotteryNumbers = new TreeSet<>(lotteryNumbers);
     }
 
     public LotteryTicket(int[] numbers) {
-//        this.lotteryNumbers = new ArrayList<>();
-        this.lotteryNumbers = new TreeSet<>();
-        LotteryNumber lotteryNumber;
-        for(int number : numbers) {
-            lotteryNumber = new LotteryNumber(number);
-            this.lotteryNumbers.add(lotteryNumber);
-        }
-        if (lotteryNumbers.size() != NUMBER_OF_LOTTERY_NUMBERS) {
-            throw new IllegalArgumentException("중복없는 6개의 숫자가 입력되어야 합니다.");
-        }
-//        Collections.sort(this.lotteryNumbers);
+        this(Arrays.stream(numbers).mapToObj(number -> new LotteryNumber(number)).collect(Collectors.toList()));
```

Comment:
```suggestion
public LotteryTicket of(int[] numbers) {
        List<LotteryNumber> lotteryNumbers = Arrays.stream(numbers)
                .mapToObj(LotteryNumber::new)
                .collect(Collectors.toList());
        return new LotteryTicket(lotteryNumbers);
    }
```
생성자를 호출하여 LotteryTicket 을 만들어 주고 계시는데 Stream은 기능단위로 줄바꿈하여 사용하는것을 권장 드려요. 
람다식 대신 메서드 레퍼런스를 활용할 수 있다면 가독성이 더욱 좋아진답니다 ㅎㅎ 
https://jinseongsoft.tistory.com/208

추가적으로 생성자가 많아지거나 객체 생성시 특별한 의미를 부여하고자 하는경우 정적 팩토리 메서드 패턴을 활용할 수 있습니다. 
참고해주시면 좋을것 같아요 ! 
https://johngrib.github.io/wiki/static-factory-method-pattern/

Reply:
정적팩토리 사용의로 적용해보겠습니다. `of` 이런게 이런 의미였군요. 또하나 배워갑니다.